packages/config/config/devices/0x0115/zme_06433.json

```diff
@@ -65,10 +66,9 @@
 		"2": {
 			"label": "Automatically switch off after",
 			"description": "If not zero, automatically switch off after a user defined time",
-			"unit": "Sec",
 			"valueSize": 2,
 			"minValue": 0,
-			"maxValue": 65535,
```

Comment:
And this doesn't seem correct either. The param is used to set the interval. The options are what is wrong.
--> remove options, set "allowManualEntry" to true.

Reply:
I've changed this back to 65535, verified in the manual. I also added a few missing parameters.